client/my-sites/stats/grid-layout.scss

```diff
@@ -3,51 +3,127 @@
-			"country country country country country country country country"
-			"authors authors authors authors search search search search"
-			"clicks clicks clicks clicks videos videos videos videos"
-			"downloads downloads downloads downloads downloads downloads downloads downloads"
-			"emails-open emails-open emails-open emails-open emails-open emails-open emails-open emails-open";
-
+		grid-template-areas: $grid-template-areas;
 	}
+}
 
-	// Mobile
-	@media ( max-width: $break-medium ) {
+@mixin stats-mobile-grid($grid-template-areas ) {
+	@media (max-width: $break-medium ) {
 		grid-template-columns: repeat(4, 1fr);
 		grid-template-rows: repeat(9, min-content);
 		gap: 0;
-		grid-template-areas:
-			"posts posts posts posts"
-			"referrers referrers referrers referrers"
-			"country country country country"
-			"authors authors authors authors"
-			"search search search search"
-			"clicks clicks clicks clicks"
-			"videos videos videos videos"
-			"downloads downloads downloads downloads"
-			"emails-open emails-open emails-open emails-open";
+		grid-template-areas: $grid-template-areas;
+	}
+}
+
+
+.stats__module-list.stats__module--unified.is-insights-page-enabled {
+	@include stats-desktop-grid(
+		"tags-categories tags-categories tags-categories tags-categories tags-categories tags-categories comments comments comments comments comments comments"
+		"shares shares shares shares followers followers followers followers publicize publicize publicize publicize"
+		"latest-post-summary latest-post-summary latest-post-summary latest-post-summary latest-post-summary latest-post-summary social-followers social-followers social-followers social-followers social-followers social-followers"
+	);
 
+	@include stats-tablet-grid(
+		"tags-categories tags-categories tags-categories tags-categories comments comments comments comments"
+		"shares shares shares shares followers followers followers followers"
+		"publicize publicize publicize publicize latest-post-summary latest-post-summary latest-post-summary latest-post-summary"
+		"social-followers social-followers social-followers social-followers social-followers social-followers social-followers social-followers"
+	);
+
+	@include stats-mobile-grid(
+		"tags-categories tags-categories tags-categories tags-categories"
+		"comments comments comments comments"
+		"shares shares shares shares"
+		"followers followers followers followers"
+		"publicize publicize publicize publicize"
+		"latest-post-summary latest-post-summary latest-post-summary latest-post-summary"
+		"social-followers social-followers social-followers social-followers"
+	);
+
+	@media (max-width: 660px) {
+		padding: 0 16px;
+	}
+
+	.stats__module-wrapper--tags-categories,
+	.list-tags-categories {
+		grid-area: tags-categories;
+	}
+
+	.stats__module-wrapper--comments,
+	.list-comments {
+		grid-area: comments;
 	}
 
+	.stats__module-wrapper--shares,
+	.list-shares {
+		grid-area: shares;
+	}
+
+	.stats__module-wrapper--followers,
```

Comment:
It'll be refactored to be a `StatsModule` soon, so I think I'll keep the class to prevent the layout breaks when the change happens. Do you think I should remove it for now?